kahuna/public/js/services/api/collections-api.js

```diff
@@ -127,6 +127,35 @@ collectionsApi.factory('collections',
             });
     }
 
+    function findCollectionResource(image, collectionToMatch){
+        return image.data.collections.filter(collection => {
```

Comment:
Best check if supported by the browsers we support, or polyfilled by Traceur (didn't look like it last time I looked).